Update shifts-connector-ukg-known-issues.md (#12557)
MicrosoftDocs/microsoft-365-docs: microsoft-365/frontline/shifts-connector-ukg-known-issues.md

The correct, expected behavior is a green check mark to indicate that the coworker accepted the request and a red "X" to indicate that the manager denied the request.

> ## A user B can’t perform some actions in Shifts after another user has signed in to Shifts through the same device

Reviewer: ```suggestion
## A user can’t perform some actions in Shifts after another user has signed in to Shifts using the same device
```

> This issue can occur if multiple users, using the same device, try to connect to Teams Shifts app and require to single sign-on (SSO) in UKG Dimensions. 

Reviewer: ```suggestion
This issue can occur if multiple users use the same device to connect to the Teams Shifts app and require a single sign-on (SSO) in UKG Dimensions. 
```

> For example, a user A signs in to Teams, requests a time off request, and signs out. To perform the request, user A was required to introduce their AAD credentials to verify (SSO). Using the same device, user B signs in to Teams and tries to perform another action in Shifts that requires SSO.

Reviewer: ```suggestion
For example, user A signs in to Teams, requests a time off request, and signs out. To perform the request, user A was required to introduce their Azure AD credentials to verify (SSO). Using the same device, user B signs in to Teams and tries to perform another action in Shifts that requires SSO.
```

> In this scenario, an issue occurs in which the user B is signed in to Teams and Shifts but user A is still signed in to UKG Dimensions.

Reviewer: ```suggestion
In this scenario, an issue occurs in which user B is signed in to Teams and Shifts but user A is still signed in to UKG Dimensions.
```

To mitigate this issue, do one of the following actions:

> - Close all the opened browser windows after each user signs out of Teams web app. This needs to happen independently of whether the user has accessed Teams web app through InPrivate window in Microsoft Edge or in Incognito mode in Google Chrome.

Reviewer: ```suggestion
- Close all the opened browser windows after each user signs out of the Teams web app. This needs to happen independently of whether the user has accessed the Teams web app through the InPrivate window in Microsoft Edge or in Incognito mode in Google Chrome.
```

> - Clear cookies and site data for the mykronos.com site in the browser after each user signs out of Teams web app. To learn more, see [Delete cookies in Microsoft Edge](https://support.microsoft.com/microsoft-edge/delete-cookies-in-microsoft-edge-63947406-40ac-c3b8-57b9-2a946a29ae09) or [Clear, enable, and manage cookies in Chrome](https://support.google.com/chrome/answer/95647).

Reviewer: ```suggestion
- Clear cookies and site data for the mykronos.com site in the browser after each user signs out of the Teams web app. To learn more, see [Delete cookies in Microsoft Edge](https://support.microsoft.com/microsoft-edge/delete-cookies-in-microsoft-edge-63947406-40ac-c3b8-57b9-2a946a29ae09) or [Clear, enable, and manage cookies in Chrome](https://support.google.com/chrome/answer/95647).
```

> - Clear cookies and site data for the mykronos.com site in the browser after each user signs out of Teams web app. To learn more, see [Delete cookies in Microsoft Edge](https://support.microsoft.com/microsoft-edge/delete-cookies-in-microsoft-edge-63947406-40ac-c3b8-57b9-2a946a29ae09) or [Clear, enable, and manage cookies in Chrome](https://support.google.com/chrome/answer/95647).

Reviewer: ```suggestion
- Clear cookies and site data for the mykronos.com site in the browser after each user signs out of the Teams web app. To learn more, see [Delete cookies in Microsoft Edge](https://support.microsoft.com/microsoft-edge/delete-cookies-in-microsoft-edge-63947406-40ac-c3b8-57b9-2a946a29ae09) or [Clear, enable, and manage cookies in Chrome](https://support.google.com/chrome/answer/95647).
```
